cocos/platform/linux/CCDevice-linux.cpp

```diff
@@ -346,9 +346,9 @@ class BitmapDC
         std::transform(lowerCasePath.begin(), lowerCasePath.end(), lowerCasePath.begin(), ::tolower);
         if ( lowerCasePath.find(".ttf") != std::string::npos ) {
             fontPath = cocos2d::FileUtils::getInstance()->fullPathForFilename(fontPath);
-            auto* fileStream = cocos2d::FileUtils::getInstance()->openFileStream(fontPath, FileStream::Mode::READ);
+            auto fileStream = cocos2d::FileUtils::getInstance()->openFileStream(fontPath, FileStream::Mode::READ);
             if ( fileStream ) {
-                delete fileStream;
+                fileStream = nullptr;
```

Comment:
redundant code since unique_ptr auto manage memory

Reply:
Oh I left that in there because it operates the same as the original non-filestream code, which was freeing the `FILE*` if the conditional result is true, so fileStream is null after the conditional block is finished.  Assuming no more code will go after the `if` conditional block, I'll remove the `fileStream = nullptr;`